.gitignore

```diff
@@ -4,3 +4,4 @@ pkg/*
 .DS_Store
 Gemfile.lock
 coverage
+.idea
```

Comment:
@dimitrovv could you remove this change from this PR? Let's keep all strictly about warnings per PR review :)